src/main/webapp/index.html

```diff
@@ -1,47 +1,57 @@
 <!DOCTYPE html>
 <html>
-<head>
-    <title>WebJars Sample - Jetty WAR</title>
-    <link rel='stylesheet' href='webjars/bootstrap/2.1.1/css/bootstrap.min.css'>
-    <script type='text/javascript' src='webjars/jquery/1.8.2/jquery.min.js'></script>
-    <script type='text/javascript' src='webjars/bootstrap/2.1.1/js/bootstrap.min.js'></script>
-    <style type="text/css">
-        body {
-            margin-top: 50px;
-        }
-    </style>
-    <script type="text/javascript">
-        $(function() {
-            $("#showModalButton").bind('click', function(event) {
-                $('#myModal').modal()
+    <head>
+        <title>WebJars Sample - Jetty WAR</title>
+        <link rel='stylesheet' href='webjars/bootstrap/${bootstrap.version}/css/bootstrap.min.css'>
```

Comment:
How do these values get populated?

Reply:
This gets populated with maven resource filter. This allow you to up the version by just changing the version number in the <properties>

<plugin>
                <groupId>org.apache.maven.plugins</groupId>
                <artifactId>maven-war-plugin</artifactId>
                <version>3.0.0</version>
                <configuration>
                    
                    <webResources>
                        <resource>
                            <directory>${basedir}/src/main/webapp</directory>
                            <filtering>true</filtering>
                            <includes>
                                <include>**/*.css</include>
                                <include>**/*.html</include>
                                <include>**/*.xhtml</include>
                            </includes>
                        </resource>
                    </webResources>
                    
                </configuration>
            </plugin>